### ISSUE TITLE
Support multiple comma-separated ESLint directives

### ISSUE THREAD

**Issue by Nasafato** [denoland/deno_lint#179]
https://github.com/denoland/deno/pull/6327#discussion_r441969628

ESLint requires multiple directives to be comma-separated, otherwise it treats the comment as one rule and throws a `not found` error. `deno lint` doesn't current handle this.

```ts
declare global {
  // eslint-disable-next-line @typescript-eslint/no-namespace
  namespace Deno {
    // eslint-disable-next-line @typescript-eslint/no-explicit-any, no-var
    var core: any;
  }
}
```

Looking in the linter code base, will need to update `parse_ignore_comment` to correctly parse the comma-separated rules.

**Comment by kitsonk** [comment]
I just ran across this in the `std` code base, trying to add two rules for a particular file, `deno lint` fails to handle it properly.
